.github/workflows/release.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     steps:
       - name: Get auth token
         id: token
-        uses: actions/create-github-app-token@af35edadc00be37caa72ed9f3e6d5f7801bfdf09 # v1.11.7
+        uses: actions/create-github-app-token@d72941d797fd3113feb6b93fd0dec494b13a2547 # v1.12.0
         with:
```

Comment:
@jeffrey-sentry  - Should we be adding any `permission-` settings here with this new version? (asking due to the changelog entry in this update)

Reply:
Should be fine, it's for supporting custom permissions that we don't use here. 
https://github.com/actions/create-github-app-token/issues/3